app/assets/javascripts/select-kit/addon/components/bulk-select-topics-dropdown.js

```diff
@@ -28,27 +28,116 @@ export default DropdownSelectBoxComponent.extend({
         name: i18n("topic_bulk_actions.update_category.name"),
         description: i18n("topic_bulk_actions.update_category.description"),
       },
+      {
+        id: "update-notifications",
+        icon: "d-regular",
+        name: i18n("topic_bulk_actions.update_notifications.name"),
+        description: i18n(
+          "topic_bulk_actions.update_notifications.description"
+        ),
+      },
+      {
+        id: "reset-bump-dates",
+        icon: "anchor",
+        name: i18n("topic_bulk_actions.reset_bump_dates.name"),
+        description: i18n("topic_bulk_actions.reset_bump_dates.description"),
+      },
+      {
+        id: "defer",
+        icon: "circle",
+        name: i18n("topic_bulk_actions.defer.name"),
+        description: i18n("topic_bulk_actions.defer.description"),
+        visible: ({ currentUser }) => currentUser.user_option.enable_defer,
+      },
       {
         id: "close-topics",
```

Comment:
When I tested I realised it might be nice to have Open Topics too, since you can close them but not reverse this. Did we used to have this bulk action? If not is it easy to add?

Reply:
No problem, it just struck me as odd we couldn't do the opposite :+1: